spec/unit/network/http/api_spec.rb

```diff
@@ -47,7 +47,9 @@ def respond(text)
       req = Puppet::Network::HTTP::Request.from_hash(:path => "/unknown")
       res = {}
       handler.process(req, res)
+      res_body = JSON(res[:body])
```

Comment:
Generally not a good idea to use `JSON` as a method like this.  Are you trying to serialize to JSON, or deserialize from JSON?  I suggest changing to `JSON.parse` to be a bit more clear and also make it easier to intercept the explicit message sent to JSON, rather than an implicit message.